legion/jupyterlab-plugin/jupyter_legion/handlers/base.py

```diff
@@ -0,0 +1,28 @@
+#
+#    Copyright 2019 EPAM Systems
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+import json
```

Comment:
Rename `Deployments` to `Name`
![Screenshot from 2019-05-29 21-14-47](https://user-images.githubusercontent.com/6774206/58577139-e3abdb80-8256-11e9-81e3-0e5f5021df26.png)

Reply:
I'll do it

Reply:
wrong